specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-04-01-preview/TenantActivityLogAlerts.json

```diff
@@ -0,0 +1,547 @@
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AlertRuleLeafCondition"
+        }
+      ],
+      "properties": {
+        "anyOf": {
+          "title": "An Activity Log Alert rule 'anyOf' condition.",
+          "description": "An Activity Log Alert rule condition that is met when at least one of its member leaf conditions are met.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AlertRuleLeafCondition"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "AlertRuleLeafCondition": {
+      "description": "An Activity Log Alert rule condition that is met by comparing the field and value of an Activity Log event.\nThis condition must contain 'field' and either 'equals' or 'containsAny'.",
+      "type": "object",
+      "properties": {
+        "field": {
+          "description": "The name of the Activity Log event's field that this condition will examine.\nThe possible values for this field are (case-insensitive): 'resourceId', 'category', 'caller', 'level', 'operationName', 'resourceGroup', 'resourceProvider', 'status', 'subStatus', 'resourceType', or anything beginning with 'properties'.",
+          "type": "string"
+        },
+        "equals": {
+          "description": "The value of the event's field will be compared to this value (case-insensitive) to determine if the condition is met.",
+          "type": "string"
+        },
+        "containsAny": {
+          "description": "The value of the event's field will be compared to the values in this array (case-insensitive) to determine if the condition is met.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ActionList": {
+      "type": "object",
+      "description": "A list of Activity Log Alert rule actions.",
+      "properties": {
+        "actionGroups": {
+          "description": "The list of the Action Groups.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ActionGroup"
+          },
+          "x-ms-identifiers": [
+            "actionGroupId"
+          ]
+        }
+      }
+    },
+    "ActionGroup": {
+      "type": "object",
+      "description": "A pointer to an Azure Action Group.",
+      "properties": {
+        "actionGroupId": {
+          "description": "The resource ID of the Action Group. This cannot be null or empty.",
+          "type": "string"
+        },
+        "webhookProperties": {
+          "type": "object",
+          "description": "the dictionary of custom properties to include with the post operation. These data are appended to the webhook payload.",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "actionProperties": {
+          "type": "object",
+          "additionalProperties": {
```

Comment:
>additionalProperties [](http://example.com/codeflow?start=11&length=20)

Please schematize this if possible.

Reply:
Again - aligning with ActivityLogAlerts

Reply:
Is this an existing public REST API you're referring to? Link? @meydand

Reply:
I see, referenced above.